boa_tester/src/main.rs

```diff
@@ -368,13 +406,22 @@ enum TestOutcomeResult {
     Panic,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
```

Comment:
Being able to copy this around should make it easier to compare later without needing to use reference comparison and so on.

```suggestion
#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq)]
```